src/pages/members/index.js

```diff
@@ -113,7 +115,32 @@ class Members extends React.PureComponent {
       };
     });
   };
-
+  /**
+   * Sort the member so that leader should be on top
+   */
+  sortMembers = (a, b) => {
```

Comment:
this doesn't need to be a class function, could you move it out of the class?

Reply:
yes sure